packages/reactive-element/src/reactive-element.ts

```diff
@@ -1288,3 +1350,15 @@ declare global {
 // TODO(justinfagnani): inject version number at build time
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 ((globalThis as any)['reactiveElementVersions'] ??= []).push('1.0.0-rc.3');
+if (DEV_MODE) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const versions = (globalThis as any)['reactiveElementVersions'];
+  if (versions.length > 1) {
+    issueWarning!(
+      `Multiple versions of \`ReactiveElement\` loaded. ` +
```

Comment:
Seems odd that we'd warn for both `LitElement` and `ReactiveElement`, but not for `lit-html`. In the end doing them all gets gets a little spammy.

Wdyt about doing the check in all three places but just warning using a generic string like `Multiple versions of Lit loaded.` so it globally dedupes, and link to a lit.dev blurb about multiple versions. The lit.dev section could cover:

1. Multiple versions should work (new feature in Lit 2)
2. We don't recommend it because while Lit is small it means you're shipping extra bytes
3. You should try `npm dedupe` and recommend components be published without bundling their dependencies and with as wide of version constraints as possible
4. You can check `window.litElementVersions`, `reactiveElementVersions`, `litHtmlVersions` to see which libraries are loaded.

Reply:
Done. Need to update lit.dev with blurb, added https://github.com/lit/lit.dev/issues/455.